### PR TITLE
adding rpm builds for ppc64le and s390x

### PR DESCRIPTION
**Pull request by utzb**
Allows to build rpms on ppc64le and s390x for users of openSUSE and CentOS/ClefOS environments. Used via build-rpm target of make.sh.

Signed-off-by: Christy Perez <christy@linux.vnet.ibm.com>
Signed-off-by: Christopher Jones <tophj@linux.vnet.ibm.com>
Signed-off-by: Dominik Dingel <dingel@de.ibm.com>
Signed-off-by: Thorsten Winkler <thorsten.winkler@de.ibm.com>
Signed-off-by: Utz Bacher <utz.bacher@de.ibm.com>

**- What I did**
Add Dockerfiles and according build/generate shell scripts

**- How to verify it**
hack/make.sh build-rpm on ppc64le/s390x

**- Description for the changelog**
Enable build of rpms for ppc64le and s390x

### CODE REVIEW

File: contrib/builder/rpm/ppc64le/centos-7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/ppc64le/generate.sh"!
+#
+
+FROM ppc64le/centos:7
+
+RUN yum groupinstall -y "Development Tools"
+RUN yum -y swap -- remove systemd-container systemd-container-libs -- install systemd systemd-libs
+RUN yum install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel libtool-ltdl-devel pkgconfig selinux-policy selinux-policy-devel sqlite-devel systemd-devel tar git cmake vim-common
+
+ENV GO_VERSION 1.7.5
```
Comment (tophj-ibm) [inline contrib/builder/rpm/ppc64le/centos-7/Dockerfile]: Can you bump all the go versions to 1.8.1? The go 1.8.1 PR got merged last night.

File: contrib/builder/rpm/ppc64le/build.sh [hunk 2]
```diff
@@ -3,6 +3,8 @@ set -e
 
 cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
+wget http://download.opensuse.org/repositories/Virtualization:/containers:/images:/openSUSE-Tumbleweed/images/openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar.xz && unxz openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar.xz && docker load -i openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar
```
Comment (tophj-ibm) [inline contrib/builder/rpm/ppc64le/build.sh]: @clnperez we don't need this right? This was just for testing tumbleweed when we didn't have an image for it?

File: contrib/builder/rpm/ppc64le/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# `dockercore/builder-rpm`
```
Comment (tophj-ibm) [inline contrib/builder/rpm/ppc64le/README.md]: You can also drop this README. Amd64 is the only other architecture to have one in all of the deb/rpms, and IMO it's not really needed or useful. Keeps consistent with s390x in this PR too :)

File: contrib/builder/rpm/ppc64le/build.sh (hunk 2 above)
Comment (clnperez) [inline contrib/builder/rpm/ppc64le/build.sh]: Yah, let's get rid of that line. I was testing with tumbleweed before I switched to the leap releases.

**Comment by utzb** [comment]
@tophj-ibm done, thanks!

**Pull request closed without merging**